HISTORY.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## 0.3.0 (2020-04-02)
+
+- Introduces IPFS based hashing of files to detect changes, ensure consistency and allow for content addressing
```

Comment:
@marcofavorito what am I missing?

Reply:
Protocol, Contract, Skill and Connection inherits from the same class, `Component`

Reply:
rephrase as necessary. Maybe not even worth citing that.

Reply:
some words on the fact that all the protocols are now generated by the generator?